src/main/java/de/qabel/desktop/ui/remotefs/RemoteFSController.java

```diff
@@ -32,10 +41,10 @@
 
 public class RemoteFSController extends AbstractController implements Initializable {
 
-    private static final String PRIVATE_KEY = "77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a";
-    public static final String ROOT_FOLDER_NAME = "root Folder";
-    private final String bucket = "qabel";
-    private final String prefix = "qabelTest";
+    private QblECKeyPair KEY_PAIR;
+    public String ROOT_FOLDER_NAME = "RootFolder";
```

Comment:
private please.
And [LCC](https://en.wikipedia.org/wiki/CamelCase#Variations_and_synonyms) for variables